pkg/network/filter/packet_source.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//nolint:revive // TODO(NET) Fix revive linter
```

Comment:
Can you add a package description? It feels bad to add a new file with a linter pragma.